sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobContainerClientExtensions.cs

```diff
@@ -88,8 +88,8 @@ public static async Task<TransferOperation> UploadDirectoryAsync(
             BlobContainerClientTransferOptions options,
             CancellationToken cancellationToken = default)
         {
-            StorageResource localDirectory = s_filesProvider.Value.FromDirectory(localDirectoryPath);
-            StorageResource blobDirectory = s_blobsProvider.Value.FromClient(client, options?.BlobContainerOptions);
+            StorageResource localDirectory = LocalFilesStorageResourceProvider.FromDirectory(localDirectoryPath);
+            StorageResource blobDirectory = BlobsStorageResourceProvider.FromClient(client, options?.BlobContainerOptions);
```

Comment:
Can delete the static members at the top of this file now.

Reply:
Done